[docs] Add kafka-api global version (#702)
ydb-platform/ydb: ydb/docs/en/core/reference/kafka-api/constraints.md

> # Constraints

Reviewer: ```suggestion
# Kafka API constraints

```

# Kafka API constraints

> 
> 

Reviewer: ```suggestion

```

> # Kafka API constraints
> 
> 
> YDB supports [Apache Kafka protocol](https://kafka.apache.org/protocol.html) version 3.4.0 with the following constraints:

Reviewer: ```suggestion
# Kafka API constraints

YDB supports [Apache Kafka protocol](https://kafka.apache.org/protocol.html) version 3.4.0 with the following constraints:

```

> YDB supports [Apache Kafka protocol](https://kafka.apache.org/protocol.html) version 3.4.0 with the following constraints:
> 
> 
> 1. Only authenticated connections are allowed.

Reviewer: ```suggestion
YDB supports [Apache Kafka protocol](https://kafka.apache.org/protocol.html) version 3.4.0 with the following constraints:

1. Only authenticated connections are allowed.

```

> The Kafka protocol version 3.4.0 is supported with limitations:

Author: > missing title
✅

> The Kafka protocol version 3.4.0 is supported with limitations:

Author: > it's better to use the same terminology in filename and text (limitations vs constraints)
✅

1. Only authenticated connections allowed.
2. Only SASL/PLAIN authentication supported.
3. Message compression not supported.
4. Only Manual Partition Assignment is supported in read mode, [assign method](https://kafka.apache.org/35/javadoc/org/apache/kafka/clients/consumer/KafkaConsumer.html#assign(java.util.Collection)), without using consumer group partitions.
5. Transactions not supported.

> 6. DDL transactions not supported. To perform DDL operations, use the [YDB SDK](../ydb-sdk/index.md) or [YDB CLI](../ydb-cli/index.md).

Reviewer: ```suggestion
6. DDL operations are not supported. Use the [YDB SDK](../ydb-sdk/index.md) or [YDB CLI](../ydb-cli/index.md) to perform them.

```
